src/Angor/Client/Pages/Browse.razor

```diff
@@ -325,25 +325,26 @@ else
             var primaryRelay = _NetworkService.GetPrimaryRelay();
 
             // Initialize flags to track service availability
-            bool explorerAvailable = false;
+
+            //bool explorerAvailable = false;
             bool indexerAvailable = false;
             bool relayAvailable = false;
 
             // Check if the primary explorer is available
-            if (primaryExplorer != null)
```

Comment:
I didn't even realize we are checking for explorers, we dont need that checks so good we remove it.